azure-pipelines.yml

```diff
@@ -52,16 +52,16 @@ steps:
       esac
     displayName: 'OPAM 2'
 
-  - script: |
-      opam install ocamlformat
-      make rule-check
-    displayName: 'Rule check'
-
   - script: |
       ./opam2_local switch create ocaml-base-compiler.4.06.1
       eval `./opam2_local config env`
     displayName: 'OCaml compiler'
 
+  - script: |
+      ./opam2_local install ocamlformat
```

Comment:
By a separate step, I meant it should be in its own `script` section, i.e.
```yaml
- script: |
     ./opam2_local install ocamlformat
  displayName: OCamlformat
```

Reply:
Okay will do this.